_pages/algorithm-development/ci-cd.md

```diff
@@ -127,3 +127,69 @@ To start using this in your own model development repositories, we have two full
 
 For more details on the configuration and capabilities of our GitHub Action, check out our detailed documentation on the GitHub Actions marketplace.
 <a href="https://github.com/algorithmiaio/algorithmia-modeldeployment-action" class="btn btn-default btn-primary"><i class="fa fa-github" aria-hidden="true"></i> TRY IT OUT: GitHub Actions for Model Training Repo</a>
+
+
+
+
+## GitLab CI/CD
+
+To deploy your algorithm files to Algorithmia everytime you push to your GitLab repository, make sure you have the following configured. 
+
+### Algorithm Files
+Similar to your default algorithm repo template, make sure your `src` folder and `requirements.txt` are in the root level of the repository. 
+
+### Gitlab CI file and Algorithmia CI scripts
+To get started with your pipelines, make sure you have the following at the root level of your repository:
+- `.gitlab-ci.yml` file 
+- `ci_entrypoint.py` file, as the entrypoint to your Algorithmia Deployment CI jobs
+- `algorithmia_ci` module directory, containing the scripts for deploying to Algorithmia and publishing a new version after runnning your defined tests.
+
+### Gitlab CI Environment Variables
+Go to your repository's Settings -> CI/CD page and expand the Variables section. Add the following CI/CD variables so that your pipeline job scripts can access these values in a secured manner.
```

Comment:
```suggestion
Go to your repository's **Settings** -> **CI/CD** page and expand the **Variables** section. Add the following CI/CD variables so that your pipeline job scripts can access these values in a secure manner.
```

Reply:
I've been **bolding** names like this so that they pop out when reading quickly through the docs.

Reply:
that's a good suggestion to keep in mind!